AutomationFramework/src/test/java/vk/steps/MainPageSteps.java

```diff
@@ -0,0 +1,56 @@
+package vk.steps;
+
+import framework.utils.GetProperties;
+import io.restassured.RestAssured;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.json.JSONObject;
+import vk.pages.MainPage;
+
+public class MainPageSteps {
+    private String ownerId = GetProperties.getProperties("vkApi", "owner_id");
+    private String accessToken = GetProperties.getProperties("vkApi", "access_token");
+    private String vkApiVersion = GetProperties.getProperties("vkApi", "v");
+    private String postThisTextOnTheWall = GetProperties.getProperties("vkApi", "message");
+    MainPage mainPage = new MainPage();
+
+    public void authorization(String email, String password) {
+        mainPage.enterEmail(email);
+        mainPage.enterPasswordAndContinue(password);
+    }
+
+    public String createPost() {
+        String postId;
+        RestAssured.baseURI = "https://api.vk.com/method/wall.post?";
+        RequestSpecification httpRequest = RestAssured.given();
+        httpRequest.param("owner_id", ownerId);
```

Comment:
using row http requsts in steps is bad, you need to create requests and controller classes with complete methods, which will accept only needed parameters, like sendMessage("userId", "message text"), not sendMessage(token, version, ...)
You can look for examples here https://habr.com/ru/post/435144/